asabiris/orchestration/kafkahandler.py

```diff
@@ -115,9 +116,8 @@ async def send_email(self, json_data):
 			attachments=json_data.get("attachments", []),  # Optional
 		)
 
-
 	async def send_to_slack(self, msg):
 		# TODO: This ... based on send_email() method
-		template = msg.pop("template")
-		body = await self.JinjaFormatterService.format(msg, template)
+		body = msg.pop("body")
+		body = await self.JinjaService.format(body['template'], body['params'])
 		await self.App.SlackOutputService.send(body)
```

Comment:
The correct name of `App.SlackOutputService` should be `App.SendSlackOrchestrator` (following `App.SendMailOrchestrator`)

Reply:
@ateska Are you sure?

it's a belongs to outputservice
https://github.com/TeskaLabs/asab-iris/blob/main/asabiris/output/slack/service.py

Reply:
It should be modelled in a same way as "send mail" part ... for a consistency.

Reply:
@ateska Got it!